README.md

```diff
@@ -51,6 +51,10 @@ this.logger.log({ id: 'your message here' }, 'context-name');
 
 ```
 
+### Authentication
+The Master Data Management API requires an API Key in order to access its resources.
+This can be achieved by providing a randomised API Key as an environment variable (`API_KEY`) and a strategy (`API_KEY_STRATEGY`) which defines the name of the header passed to the API
```

Comment:
Can we have a full-stop at the end of this line please?

Reply:
updated now